examples/acl/project.clj

```diff
@@ -1,8 +1,7 @@
 (defproject acl "0.1.0-SNAPSHOT"
   :description "FIXME: write description"
   :min-lein-version "2.0.0"
-  :dependencies [[org.clojure/clojure "1.10.3"]
-                 [com.flexiana/framework "0.4.0-rc5"]]
+  :dependencies [[com.flexiana/framework "0.5.0-rc1"]]
```

Comment:
why is clojure removed from the dependencies? we are directly using it in the project, but this way clojure is only transitive dependency brought in by framework. In my opinion, this should be avoided.

Reply:
I agree that usually it's preferable to have explicit dependency on clojure in a project. The main reasoning behind this was avoiding repetition (necessity to change clojure version in all examples each time we update it in the framework). Considering it works I would prefer to leave it as is.

Only downsides of relying on transitive dependency I see atm is the possibility of the dependency being removed from an intermediate project. But this should be pretty easily fixable on the compilation stage. And it's not like we have plans to remove clojure deps from framework.
If you have any other arguments pls let me know.